### PR TITLE
dev guide release post, 0.7.0

### PR DESCRIPTION
**Pull request by maelle**
TODOS

- [x] Change dev guide links to devguide.ropensci.org

### CODE REVIEW

File: content/blog/2021-11-18-devguide-0.7.0/index.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+As usual resources and a close-captioned recording will be posted afterwards.
+
+## Still some work for editors 😉
+
+Fear not, software peer review volunteer [editors](https://ropensci.org/software-review/) still participate a lot in the process.
+We've formalized [editor responsabilities](https://devdevguide.netlify.app/editorguide.html#editors-responsabilities), in particular outlining editor etiquette around submissions handled by other editors.
+
+> You do not have to keep track of other submissions, but if you do notice an issue with a package that is being handled by another editor, feel free to raise that issue directly with the other editor, or post the concern to editors-only channel on slack. 
+
+Thanks to the new statistical software peer-review editor [Toby Dylan Hocking](https://github.com/tdhock) we've made it explicit that editors have to look for __two__ reviewers.
```
Comment (mpadge) [inline content/blog/2021-11-18-devguide-0.7.0/index.md:49]: Ping @tdhock here, by way of gratitude for his assistance :+1:

Comment (maelle) [reply content/blog/2021-11-18-devguide-0.7.0/index.md:49]: Indeed, thanks again!

File: content/blog/2021-11-18-devguide-0.7.0/index.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+We are thrilled to remove some workload from reviewers as we are aware that reviewing a package is a lot of work. 
+Thanks to all reviewers for their incredible contributions to Software Peer Review!
+
+## Miscellaneous guidance updates
+
+The requirement to have a vignette is now more precise, there has to be at least one _HTML_ vignette. 
+That requirement is part of pkgcheck checks.
+
+Thanks to an idea of editor [Anna Krystalli](https://github.com/annakrystalli), the dev guide now includes an explicit [subsection about version control](https://devdevguide.netlify.app/building.html#version-control), and pkgcheck's checks include looking for "scrap files" such as `.DS_Store` files, that should be gitignored.
```
Comment (noamross) [inline content/blog/2021-11-18-devguide-0.7.0/index.md:93]:
```suggestion
Thanks to an idea of editor [Anna Krystalli](https://github.com/annakrystalli), the dev guide now includes an explicit [subsection about version control](https://devdevguide.netlify.app/building.html#version-control), and `pkgcheck`'s checks include looking for "scrap files" such as `.DS_Store` files, that should be `.gitignore`'d.
```

File: content/blog/2021-11-18-devguide-0.7.0/index.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+* In the packaging guide, we now give precise examples of [tools for installation instructions](https://devdevguide.netlify.app/building.html#readme): the r-lib remotes and pak packages but also [R-universe](/r-universe/)!
+
+## Setting up your package to foster a community
+
+Following our community call ["Set Up Your Package to Foster a Community"](/commcalls/apr2021-pkg-community/) that has been [summarized by  Sina Rüeger](/blog/2021/04/28/commcall-pkg-community/), we've added guidance to our [collaboration guide](https://devdevguide.netlify.app/collaboration.html).
+
+## Updates to reviewer guidance
+
+The work on pkgcheck and buffy means we were able to remove some elements from reviewers' guidance:
```
[the remark this answers is not in the document]
Comment (maelle) [reply content/blog/2021-11-18-devguide-0.7.0/index.md:81]: I've opened an issue in the blog guide https://github.com/ropensci-org/blog-guidance/issues/184

File: content/blog/2021-11-18-devguide-0.7.0/index.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
+---
+title: "rOpenSci Dev Guide 0.6.0: Updates"
```
Comment (steffilazerte) [inline content/blog/2021-11-18-devguide-0.7.0/index.md:2]: :grin: 
```suggestion
title: "rOpenSci Dev Guide 0.7.0: Updates"
```

File: content/blog/2021-11-18-devguide-0.7.0/index.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+
+* In the packaging guide, we now give precise examples of [tools for installation instructions](https://devdevguide.netlify.app/building.html#readme): the r-lib remotes and pak packages but also [R-universe](/r-universe/)!
+
+## Setting up your package to foster a community
+
+Following our community call ["Set Up Your Package to Foster a Community"](/commcalls/apr2021-pkg-community/) that has been [summarized by  Sina Rüeger](/blog/2021/04/28/commcall-pkg-community/), we've added guidance to our [collaboration guide](https://devdevguide.netlify.app/collaboration.html).
+
+## Updates to reviewer guidance
+
+The work on pkgcheck and buffy means we were able to remove some elements from reviewers' guidance:
```
Comment (steffilazerte) [inline content/blog/2021-11-18-devguide-0.7.0/index.md:85]: What is buffy? Perhaps add a footnote? You talk about pkgcheck above, so I think that's clear.

File: content/blog/2021-11-18-devguide-0.7.0/index.md [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+The requirement to have a vignette is now more precise, there has to be at least one _HTML_ vignette. 
+That requirement is part of pkgcheck checks.
+
+Thanks to an idea of editor [Anna Krystalli](https://github.com/annakrystalli), the dev guide now includes an explicit [subsection about version control](https://devdevguide.netlify.app/building.html#version-control), and pkgcheck's checks include looking for "scrap files" such as `.DS_Store` files, that should be `.gitignore`'d.
+
+We've added some more guidance on [making packages easier to cite](https://devdevguide.netlify.app/building.html#version-control) following our [blog post on the same topic](/blog/2021/02/16/package-citation/).
+
+## Policy update: scope clarification for data extraction and munging
+
+We've clarified the definition of the [categories](https://devdevguide.netlify.app/policies.html#package-categories) " data extraction" and "data munging" by adding examples.
```
Comment (steffilazerte) [inline content/blog/2021-11-18-devguide-0.7.0/index.md:105]:
```suggestion
We've clarified the definition of the [categories](https://devdevguide.netlify.app/policies.html#package-categories) "data extraction" and "data munging" by adding examples.
```

**Review by steffilazerte — COMMENTED** [review]
Lovely overview and really great work adding in all the references to other rOpenSci projects and blog posts, I love how it ties everything together. 

I have three itty bitty suggested changes and one request for an author file (Emily doesn't have one!), other than that it looks great!

* [X] post follows Content Guidelines
* [ ] post follows Style Guide
* [X] title is in Title Case
* [X] publication date is ok
* [X] alternative text of images is informative
* [X] Twitter metadata looks ok (paste post preview link in [Twitter card validator](https://cards-dev.twitter.com/validator); might have to click twice on "Preview card")
* [ ] author metadata is provided with correct folder name
* [X] html not included in pull request of Rmd post
* [X] I ran `roblog::ro_lint_md()` on index.md
* [X] I ran `roblog::ro_check_urls()` on index.md
* [X] I ran a spell-check on index.md
* [X] YAML subject tags are ok ("tech notes" for tech notes; "community" for non-staff non-editor)

File: content/blog/2021-11-18-devguide-0.7.0/index.md (hunk 5 above)
Comment (maelle) [reply content/blog/2021-11-18-devguide-0.7.0/index.md]: I rephrased, good point!

**Comment by maelle** [comment]
@steffilazerte thanks a lot for your review & the kind words! Changes made :heavy_check_mark:

**Comment by maelle** [comment]
Note to self: do not forget to update links to the dev guide prod version once it is released!

**Pull request merged**